frontend/amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -368,14 +368,25 @@ export class TableDetail extends React.Component<
     const databricksApps = apps.filter(
       (app) => app.name.toLowerCase() === DATABRICKS.toLowerCase()
     );
+    // <Brex>
+    const customApplications = [
+      'feature_link',
+      'snowflake_link',
+      'postgres_link',
+      'plan',
+    ];
+    // </Brex>
     const remainingApps = apps.filter(
       (app) =>
         app.name.toLowerCase() !== AIRFLOW.toLowerCase() &&
-        app.name.toLowerCase() !== DATABRICKS.toLowerCase()
+        app.name.toLowerCase() !== DATABRICKS.toLowerCase() &&
+        // <Brex>
+        customApplications.indexOf(app.name.toLowerCase()) === -1
+      // </Brex>
     );
 
     return (
-      <div>
+      <>
```

Comment:
does it still need to be a <div> or doesn't matter?

Reply:
I tied to keep `<div>` in the beginning but it turns out that buttons listed inside this `<div>` are not aligned with outside buttons `Freshness` and `Preview`, hence removing